distribution/docs/src/main/resources/_contents/_tables/org.codice.ddf.distribution.landing-page.properties-table-contents.adoc

```diff
@@ -0,0 +1,75 @@
+.[[org.codice.ddf.distribution.landing-page.properties]]Landing Page
+[cols="1,1m,1,3,1,1" options="header"]
+|===
+
+|Name
+|Id
+|Type
+|Description
+|Default Value
+|Required
+
+|Description
+|description
+|String
+|Specifies the description to display on the landing page.
+|As a common data layer\, DDF provides secure enterprise-wide data access for both users and systems.
```

Comment:
I didn't use ${branding} here because this file was generated by a script originally, but I've added it now.